apps/src/p5lab/P5Wrapper.js

```diff
@@ -1,12 +1,20 @@
+import p5 from '@code-dot-org/p5';
+
 import {getStore} from '@cdo/apps/redux';
+
+// Disabling import order in order to maintain p5 import first.
+// This might be safe to remove but needs investigation whether any behavior is changed by order.
```

Comment:
`This might be safe to remove but needs investigation whether any behavior is changed by order.`

With such a big PR I want to minimize the number of possible unintended side effect and changing when window.p5 is defined could be a side effect.

Reply:
Ahh, that makes sense. Could you update the comment to highlight the `window.p5` assignment rather than just the importing of p5?